### ISSUE TITLE
I have hakchi2-ce it modded my nes mini classic  but I have a snes mini authentic non-modded and I get a error.

### ISSUE THREAD

**Issue by valdor1** [TeamShinkansen/Hakchi2-CE#362]
I get a error when I try yo mod mt snes mini. The error says cant open device when try to flash the kernel. Any ideas why?

**To Reproduce**  

**Expected behavior**  

**Screenshots**  

**Additional context**

**Comment by DanTheMan827** [comment]
Try again, that error can happen randomly

**Comment by valdor1** [comment]
I uninstalled hakchi2 and reinstalled tried several times get same error.
Nes mini modded great.  Snes just isnt working for me.

**Comment by ReyVGM** [comment]
You have a fake mini or are using the wrong USB cable.

**Comment by PokeyManatee4** [comment]
Yeah Prob Fake Just Open To See That's How I Verify Mines Was Real And Youtube Videos That Did the same also
